addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js

```diff
@@ -13,6 +13,7 @@ import {
 } from "@odoo/owl";
 import { useService } from "@web/core/utils/hooks";
 import dom from "@web/legacy/js/core/dom";
+import { preserveCursor } from "@web_editor/js/editor/odoo-editor/src/OdooEditor";
```

Comment:
Any reason to not import it from the utils (`@web_editor/js/editor/odoo-editor/src/utils/utils`) ? I feel it may be clearer :+1:

Reply:
No specific reason, I just changed it according to review given above. rolling back to first one. ; )